src/Raven.Server/Documents/Handlers/MultiGetHandler.cs

```diff
@@ -188,7 +190,7 @@ public async Task PostMultiGet()
                         writer.WriteEndObject();
                         writer.WriteEndObject();
                         if (TrafficWatchManager.HasRegisteredClients)
```

Comment:
Check `sb != null` instead, to avoid concurrency issues.

Reply:
done